src/platform/common/stm32/bus_spi_hw.c

```diff
@@ -172,12 +172,17 @@ FAST_IRQ_HANDLER static void spiTxIrqHandler(dmaChannelDescriptor_t* descriptor)
 }
 #endif
 
+uint16_t tmp_divisor;
+
+
```

Comment:
tmp_divisor is not being used ?
```suggestion
```

Reply:
Yeah spotted that and was already rebasing!